demisto_sdk/commands/format/update_generic.py

```diff
@@ -239,6 +240,13 @@ def set_fromVersion(self, from_version=None, file_type: Optional[str] = None):
             elif should_set_from_version:
                 if self.data.get(self.from_version_key) != '5.5.0' or file_type != INTEGRATION:
                     self.data[self.from_version_key] = VERSION_6_0_0
+            # If it is new pack, and it has from version lover than 5.5.0, ask to set it to 5.5.0
+            elif self.data.get(self.from_version_key) in ['5.0.0', '4.5.0'] and file_type != PLAYBOOK:
```

Comment:
do not check for only 5.0.0 and 4.5.0 - use `LooseVersion` and check lower than 5.5.0
Also why the exception for playbooks?

Reply:
LooseVersion - done.
Why the exception for playbooks? at this file:  demisto_sdk/commands/format/update_playbook.py we have function: def update_fromversion_by_user(). If there is no Fromversion, this function already asks user which fromversion to add.
If user will add fromversion lover than 5.5.0, and there will not be "file_type != PLAYBOOK" check, we will ask the user if he want to update it.. It can be strange. What do you prefer to do? Ask him again if he sure that he wants version lower that 5.5.0?